start.py

```diff
@@ -19,6 +19,7 @@
 import datadog  # noqa: E402
 import instadeploy  # noqa: E402
 import requests  # noqa: E402
+import metrics   # noqa: E402
```

Comment:
Metrics is part of the buildpack, so should be grouped with the 3rd group of imports (line 24 onwards), as per [PEP 8](https://www.python.org/dev/peps/pep-0008/?#imports)

Reply:
No longer relevant